app/models/spree/payment_method/sisow_billing/sisow_payment_method.rb

```diff
@@ -0,0 +1,28 @@
+module Spree
+  class PaymentMethod::SisowBilling::SisowPaymentMethod < PaymentMethod
+    def payment_profiles_supported?
+      false
+    end
+
+    def auto_capture?
+      true
+    end
+
+    def purchase(amount, source, opts)
```

Comment:
Unused method argument - `amount`. If it's necessary, use `_` or `_amount` as an argument name to indicate that it won't be used.<br>Unused method argument - `opts`. If it's necessary, use `_` or `_opts` as an argument name to indicate that it won't be used.